mmv1/third_party/terraform/tests/resource_data_loss_prevention_deidentify_template_test.go

```diff
@@ -1369,6 +1418,56 @@ resource "google_data_loss_prevention_deidentify_template" "config" {
         }
       }
 
+      transformations {
+        info_types {
+          name = "CRYPTO_HASH_TRANSIENT_UPDATED_EXAMPLE"
+        }
+
+        primitive_transformation {
+          crypto_hash_config {
+            crypto_key {
+              transient {
+                # update value
+                name = "beepy-beep-updated"
+              }
+            }
+          }
+        }
+      }
+
+      transformations {
+        info_types {
+          name = "CRYPTO_HASH_UNWRAPPED_UPDATED_EXAMPLE"
+        }
+
+        primitive_transformation {
+          crypto_hash_config {
+            crypto_key {
+              unwrapped {
+                key = "VVdWVWFGZHRXbkUwZERkM0lYb2xRdz09"
```

Comment:
Just want to double-check that this & the transformation below are intentionally not changing the values?

Reply:
I'm okay with merging this as-is, just wanted to make sure it's intentional. Note that this means that we're not specifically testing that this field being updated works, but we still have good coverage overall.